.pre-commit-config.yaml

```diff
@@ -8,6 +8,11 @@ repos:
     -   id: check-added-large-files
     -   id: flake8
         exclude: ^docs/conf.py$
+-   repo: https://github.com/ambv/black
+    rev: 18.6b2
```

Comment:
Is there any way to just say latest version?

Reply:
[intentionally not](https://pre-commit.com/#using-the-latest-version-for-a-repository) -- speed and repeatability trade-offs were made.

Fortunately `pre-commit autoupdate` is a quick and easy way to get to the latest :)